runtime/src/iree/base/internal/CMakeLists.txt

```
@@ -91,6 +91,44 @@ iree_cc_test(
     iree::testing::gtest_main
 )
 
+iree_cc_library(
+  NAME
+    bitmap
+  HDRS
+    "bitmap.h"
+  SRCS
+    "bitmap.c"
+  DEPS
+    iree::base
```

missing dep (via bazel_to_cmake)
```suggestion
  DEPS
    ::internal
    iree::base
```